custom/ewsghana/alerts/alerts.py

```diff
@@ -266,7 +266,7 @@ def report_reminder_process_user(user, test=False):
     now = datetime.datetime.utcnow()
     date = now - datetime.timedelta(days=7)
 
-    if not user.location or user.location.location_type not in get_reporting_types(user.domain):
```

Comment:
`get_reporting_types` returned a list of all location types with `administrative == False`.  It was only used here, so I just looked at the `administrative` property directly instead, and deleted that function.